wa/workloads/jetnews/__init__.py

```diff
@@ -0,0 +1,48 @@
+#    Copyright 2023 ARM Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from wa import Workload
+
+
+class JetNews(Workload):
+    name = "jetnews"
+    description = "Run the JetNews basic tests"
+    package_names = ["com.example.jetnews", "com.arm.benchmark"]
+
+    def setup(self, context):
+        super(JetNews, self).setup(context)
```

Comment:
There is no need to pass any parameter to `super()` in this context, we are in Python 3

Reply:
Thanks. I made this change now.